File: PWA/Application.WASM/IndexDbEntities/Message.cs
using Application.WASM.Models;

namespace Application.WASM.IndexDbEntities
{
    public class Message
    {
        [System.ComponentModel.DataAnnotations.Key]
        public long Id { get; set; }
        public DateTime CreatedDate { get; set; }
        public DateTime? LastModifiedDate { get; set; }
        public bool IsEdited { get; set; }
        public string MessageId { get; set; }
        public string Message_Id { get; set; }
         public long User_Id { get; set; }
        public string Group_Id { get; set; }
        public long? PrivateRoom_Id { get; set; }
        public string Reply_To_MessageId { get; set; }
        public string Text { get; set; }
        public string Sticker_Id { get; set; }
        public string Gif_Id { get; set; }
    }
}

File: Apps/bomapp/lib/entities/message.dart
import 'dart:ffi';

import 'package:bomapp/models/ResponseMessage.dart';
import 'package:floor/floor.dart';

@entity
class Message {
  @PrimaryKey(autoGenerate: true)
  int? id;
  int? orginalId;
  String? messageId;
  int? userId;
  int? groupId;
  int? privateId;
  int? replyMessageId;
  String? text;
  int? stickerId;
  int? gifId;

 
  Message.fromResponseMessage(ResponseMessage responseMessage) {
orginalId=responseMessage.id;
messageId=responseMessage.messageId;
userId=responseMessage.userId;
gifId=responseMessage.gifId;
groupId=responseMessage.groupId;
replyMessageId=responseMessage.replyToMessageId;
text=responseMessage.text;
stickerId=responseMessage.stickerId;
gifId=responseMessage.gifId;

  }
}
